pkg/amber/provenance_test.go

```diff
@@ -50,8 +50,8 @@ func TestExampleProvenance(t *testing.T) {
 
 	// Check that the provenance parses correctly
 	testutil.AssertEq(t, "repoURL", predicate.Materials[1].URI, "https://github.com/project-oak/oak")
-	testutil.AssertEq(t, "commitHash length", len(predicate.Materials[1].Digest["sha1"]), wantSha1HexDigitLength)
-	testutil.AssertEq(t, "builderImageID length", len(predicate.Materials[0].Digest["sha256"]), wantSha256HexDigitLength)
+	testutil.AssertEq(t, "commitHash length", len(predicate.Materials[1].Digest["sha1"]), wantSHA1HexDigitLength)
```

Comment:
Why is SHA-1 supported if it is broken?

Reply:
This is the git commit hash. I guess we could use sha256, but that would make it difficult to use since currently Git (and so Github) use SHA1 for commit hashes.

Reply:
Oh I didn't know Git used SHA1. I guess that is a valid use since git hashes are not "for security"  (and more to your point we want to be compatible with git hashes).